client/stream.cpp

```diff
@@ -73,7 +73,7 @@ void Stream::clearChunks()
 
 void Stream::addChunk(msg::PcmChunk* chunk)
 {
-	while (chunks_.size() * chunk->duration<cs::msec>().count() > 10000)
+	while (chunks_.size() * std::chrono::duration_cast<cs::usec>(chunk->duration<cs::msec>()) > bufferMs_ * 10 )
```

Comment:
While this patch more closely resembles the current approach of the Stream class, I do not feel it is correct to discard chunks in the stream class itself.

A stream should be just that. Some other entity should handle discarding of chunks. How do you folks feel about this? I would be glad to entirely remove the check